app.test.js

```diff
@@ -19,4 +19,84 @@ describe('Server', () => {
     });
   });
 
+  describe('GET /api/v1/projects/:id', () => {
+    it('should return a happy status code of 200 and a single project object if the project exists', async () => {
+      const expectedProject = await database('projects').first();
+        delete expectedProject.created_at;
+        delete expectedProject.updated_at;
+      const { id } = expectedProject;
+
+      const response = await request(app).get(`/api/v1/projects/${id}`);
+      const result = response.body[0];
+        delete result.created_at;
+        delete result.updated_at;
```

Comment:
Again, I'd love if you could explain what's happening here. I haven't seen this yet